#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("threshold_linear_x", double_t, 0, "Component wise pose error threshold allowed for linear x", 15, 0, 50)
gen.add("threshold_linear_y", double_t, 0, "Component wise pose error threshold allowed for linear y", 15, 0, 50)
gen.add("threshold_linear_z", double_t, 0, "Component wise pose error threshold allowed for linear z", 15, 0, 50)
gen.add("threshold_angular_x", double_t, 0, "Component wise pose error threshold allowed for angluar x", 15, 0, 50)
gen.add("threshold_angular_y", double_t, 0, "Component wise pose error threshold allowed for angluar x", 15, 0, 50)
gen.add("threshold_angular_z", double_t, 0, "Component wise pose error threshold allowed for angluar x", 15, 0, 50)

exit(gen.generate("mcr_geometric_relation_monitors", "mcr_geometric_relation_monitors", "ComponentWisePoseErrorMonitor"))